packages/node/src/indexer/indexer.module.ts

```diff
@@ -31,7 +36,15 @@ import { WorkerService } from './worker/worker.service';
       },
       inject: [SubqueryProject, EventEmitter2],
     },
-    DictionaryService,
+    {
```

Comment:
Does indexer.module need the dictionary service? I feel like its unnecessary

Reply:
Didn't add it, I've removed it though.